empress/__init__.py

```diff
@@ -140,6 +140,19 @@ def draw_stats(self):
         self.draw_stats_on(ax)
         return figure
 
+    def stats(self, num_trials: int = 50):
+        _, costs, p = statistics.stats(self.recon_input, self.dup_cost, self.trans_cost, self.loss_cost, num_trials)
+        return costs, p
+
+    def draw_stats_on(self, ax: plt.Axes):
+        costs, p = self.stats()
+        statistics.draw_stats(ax, self.total_cost, costs, p)
+
+    def draw_stats(self):
```

Comment:
Duplicate

Reply:
I am not sure what this is. Should I make the change?

Reply:
Removed.